admin/admin.go

```diff
@@ -128,6 +133,13 @@ type Token struct {
 	Users    []User
 }
 
+type WebAuthnToken struct {
+	CredentialName string `json:"credential_name"`
+	DateAdded      int    `json:"date_added"`
+	Label          string `json:"label"`
+	WebauthnKey    string `json:"webauthnkey"`
```

Comment:
Based on the naming pattern, should the capitalization of this field be `WebAuthnKey` ?

Reply:
Fixed! Good catch